featurebyte/models/use_case.py

```diff
@@ -30,7 +30,7 @@ class UseCaseModel(FeatureByteCatalogBaseDocumentModel):
         The default eda table id of from observation_table_ids
     """
 
-    context_id: PydanticObjectId
+    context_id: Optional[PydanticObjectId]
```

Comment:
Context API object is not ready yet. This is made optional temporarily until it is ready.